src/site/content/en/fast/optimize-css-background-images-with-media-queries/index.md

```diff
@@ -0,0 +1,148 @@
+
+In the next section, you'll learn how to apply an optimization to load different background images, according to the user's device.
+
+## Use media queries
+
+Using media queries is a common technique to declare stylesheets that will only apply to certain media or device types. They are implemented by using [@media rules](https://developer.mozilla.org/en-US/docs/Web/CSS/@media), which let you define a set of breakpoints, where specific styles are defined. 
+When the conditions defined by the `@media` rule are met (for example, a certain screen width), the group of styles defined inside the breakpoint will be applied.
+
+The following steps can be used to apply media queries to [the site](https://use-media-queries-unoptimized.glitch.me/) so that different images are used, depending on the maximum width of the device requesting the page.
+
+- In `style.css` remove the line that contains the background image URL: 
+
+```css//4
+body { 
+  background-position: center center; 
+  background-attachment: fixed; 
+  background-repeat: no-repeat; background-size: cover; 
+  background-image: url(images/background-desktop.jpg); 
+}
+```
+
+- Next, create a breakpoint for each screen width, based on the common dimensions in pixels that mobile, tablet, and desktop screens usually have:
+
+For mobile:
+
+```css
+@media (max-width: 480px) {
+    body {
+        background-image: url(images/background-mobile.jpg);
+    }
+}
+```
+
+For tablets:
+
+```css
+@media (min-width: 481px) and (max-width: 1024px) {
+    body {
+        background-image: url(images/background-tablet.jpg);
+    }
+}
+```
+
+For desktop devices:
+
+```css
+@media (min-width: 1025px) {
+    body {
+	background-image: url(images/background-desktop.jpg);
+   }
+}
+```
+
+Open the optimized version of [style.css](https://use-media-queries-optimized.glitch.me/style.css) in your browser to see the changes made.
+
+{% Aside %}
+The images used in the optimized demo are already resized to fit into different screen sizes. Showing how to resize images is out of the scope of this guide, but if you want to know more about this, the [Serve responsive images guide](https://web.dev/serve-responsive-images/) covers some useful tools, like the [sharp npm package](https://www.npmjs.com/package/sharp) and the [ImageMagick CLI](https://www.imagemagick.org/script/index.php).
+{% endAside %}
+
+## Measure for different devices
+
+Next visualize the resulting site in different screen sizes and in simulated mobile devices:
+
+1. Open the [optimized site](https://use-media-queries-optimized.glitch.me/) in a new Chrome tab.
+{% Instruction 'devtools-network', 'ol' %}
+{% Instruction 'reload-page', 'ol' %}
+Resize the browser screen to different widths and notice in the Network Log how new network requests are made as the viewport changes its width.
+
+You might have noticed that, at certain points, when the width of the browser screen is changed, new images are requested.
```

Comment:
```suggestion
When the width of the browser screen is changed, new images are requested.
```

Need to update this slightly so that it flows with the change that I'm proposing in the ordered list.